examples/parallel_simulate_evoked.py

```diff
@@ -0,0 +1,34 @@
+"""
+===============
+Simulate dipole
+===============
+
+This example demonstrates how to simulate a dipole for evoked-like
+waveforms using HNN-core.
+"""
+
+# Authors: Mainak Jas <mainak.jas@telecom-paristech.fr>
+#          Sam Neymotin <samnemo@gmail.com>
+
+import os.path as op
+
+###############################################################################
+# Let us import hnn_core
+
+import hnn_core
+from hnn_core import simulate_dipole, Params, Network, shutdown
+
+hnn_core_root = op.join(op.dirname(hnn_core.__file__), '..')
+
+###############################################################################
+# Then we read the parameters file
+params_fname = op.join(hnn_core_root, 'param', 'default.json')
+params = Params(params_fname)
+
+###############################################################################
+# Now let's simulate the dipole
+# You can simulate multiple trials in parallel by using n_jobs > 1
+net = Network(params)
+dpls = simulate_dipole(net, n_jobs=1, n_trials=4)
```

Comment:
don't we already have an example already that uses `n_trials=2`?

Reply:
Yeah, this one doesn't plot which is incompatible with MPI unless you do something like get_rank(). This script was mainly for benchmarking purposes. I think I'll remove this file from the next update to this PR

Reply:
@jasmainak I've updated this example to show the MPI invocation. Please see the new file.
https://github.com/hnnsolver/hnn-core/blob/a10123a496d105f75768c39c71917aeffe3de672/examples/parallel_simulate_evoked.py

This should be clarify how the two forms are used (for users and to your comment below)

Reply:
Note that this example has been removed.